### PR TITLE
Expose 4 more rows from the galaxy_user table to help out sys admins

### PR DESCRIPTION
**Pull request by XDtim**
Galaxy admins can use this extra data returned via the API to better automate user management and cleanup.

### CODE REVIEW

**Comment by mvdbeek** [comment]
@galaxybot test this

**Comment by XDtim** [comment]
Humm, not sure why this test case broke.  It isn't in the user part of the API.

Looking into it now

**Comment by XDtim** [comment]
Humm, I think we should re-run this.  I tested it locally on my branch with this commnd:

./run_tests.sh -api test/api/test_dataset_collections.py:DatasetCollectionApiTestCase.test_list_pair_download
And got this output:

ID: api.test_dataset_collections.DatasetCollectionApiTestCase.test_list_pair_download
Status: success

@galaxybot test this

**Comment by dannon** [comment]
@XDtim I think this test may intermittently fail.  I've restarted the test.

**Comment by mvdbeek** [comment]
Yeah, that one we should figure out or deactivate. Don't let this stop you
from merging.

**Comment by XDtim** [comment]
Should we merge this?

File: lib/galaxy/model/__init__.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ class User( object, Dictifiable ):
     histories, credentials, and roles.
     """
     # attributes that will be accessed and returned when calling to_dict( view='collection' )
-    dict_collection_visible_keys = ( 'id', 'email', 'username' )
+    dict_collection_visible_keys = ( 'id', 'email', 'username', 'deleted', 'purged', 'active', 'last_password_change' )
```
Comment (nsoranzo) [inline lib/galaxy/model/__init__.py]: I'd probably remove `'purged'`, users cannot be purged at the moment as far as I know.

Also, can you please add `'deleted', 'active', 'last_password_change'` to `dict_element_visible_keys` below? `dict_collection_visible_keys` is supposed to be a subset of it.

Comment (XDtim) [reply lib/galaxy/model/__init__.py]: I will make it so

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by jmchilton** [comment]
Thanks for the contribution - this is awesome stuff and I can definitely see this being useful to admins!

I'm a little uncomfortable with exposing these new details to non-admin users however - I don't think users should know for instance when other users last changed their passwords. I've reworked this in #4274 - if you merge my branch into this PR I will close mine - otherwise if someone does merge mine in the meantime this PR will automatically get marked as merged and your exact commits with your authorship will be included in Galaxy (it will be as if this was merged directly).

Thanks again!

**Pull request merged**